instructure/canvas-ios
# Fix Syllabus content disappearing

Always include all Course data when fetching course list to avoid Course data overwrites with the results of API calls with less includes

refs: MBL-15208
affects: Teacher, Student
release note: none

test plan:
- Enter a course which has a syllabus text content and syllabus summary as well.
- Enter Syllabus menu, observe Syllabus and Summary tabs appearing.
- Go back to Dashboard and pull to refresh. OR Go to an assignment details in the same course.
- Go back to Syllabus menu.
